apps/explorer/src/app/components/splash-loader/splash-loader.tsx

```diff
@@ -0,0 +1,32 @@
+import "./splash-loader.scss";
+
+import React from "react";
+
+export const SplashLoader = ({ text = "Loading" }: { text?: string }) => {
```

Comment:
Can we add this one to the ui toolkit? Could be a follow up if theres further setup required